scipy/io/idl.py

```diff
@@ -674,14 +674,14 @@ def __setitem__(self, key, value):
 def readsav(file_name, idict=None, python_dict=False,
             uncompressed_file_name=None, verbose=False):
     """
-    Read an IDL .sav file
+    Read an IDL .sav file.
```

Comment:
This particular one is for consistency. However, if you look at http://docs.scipy.org/doc/scipy/reference/io.html, the descriptions of `loadmat`, `whosmat`, `readsav`, `wavfile.read` and `wavfile.write` are messed up because of the missing periods. Apparently the numpy-style sphinx parser requires a period at the end of the first sentence, otherwise it just keeps going. In this case, it included parameter descriptions where they don't belong.